src/tests/sqlsmith/src/runner.rs

```diff
@@ -164,6 +365,7 @@ impl Runner {
                     tracing::info!("query_sql: {}", query_sql);
```

Comment:
we can remove this log, only print the reduced sql

Reply:
Now it is debug level.